x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-mappings.json

```diff
@@ -28,6 +28,10 @@
             "dataset": {
```

Comment:
what is dataset used for?

Reply:
It is part of the data stream naming scheme: https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme